.github/ISSUE_TEMPALTE.md

```diff
@@ -0,0 +1 @@
+This repo is OBSOLETE - please see the README file for information
```

Comment:
Do you plan to update the README as well? If so, can you include it in the PR?

Reply:
Updated the text in Readme and added @danroth27 as a reviewer.

Reply:
I worry that saying the repo is obsolete will make users think that the feature is already obsolete, even though we haven't shipped its replacement yet.

How about: "NOTE: Future enhancements for Razor compilation should now be opened in the [Razor](https://github.com/aspnet/razor) repo. Only issues (bugs) with the existing MVC precompilation feature should be opened here"